roles/rsyslog/tasks/main.yaml

```diff
@@ -1,16 +1,16 @@
 ---
 - name: Install/Update required packages
-  yum:
-    name: "{{lookup('flattened', rsyslog_base_packages, rsyslog_tls_packages if rsyslog_pki else [], rsyslog_packages)}}"
-    state: 'latest'
+  package:
+    name: "{{ lookup('flattened', rsyslog_base_packages, rsyslog_tls_packages if rsyslog_pki else [], rsyslog_packages) }}"
+    state: latest
   when:
-    - rsyslog_enabled|bool
-    - not use_rsyslog_image|default(False)|bool
+    - rsyslog_enabled | bool
+    - not use_rsyslog_image | default(false) | bool
   tags:
     - skip_ansible_lint
 
 - name: Get rsyslog version
-  yum:
+  package:
     list: rsyslog
```

Comment:
does the package module actually support this parameter? https://docs.ansible.com/ansible/latest/modules/package_module.html.

I suggest to use https://docs.ansible.com/ansible/latest/modules/package_facts_module.html , we have been using it elsewhere like here: https://github.com/linux-system-roles/timesync/commit/e145b8cda93dbbd64aba2d049ed562b29b974ae5

Reply:
Thanks!  I'm fixing it in a new PR...

Reply:
@pcahyna, I created a patch following your advice and submitted a pr: 
https://github.com/linux-system-roles/logging/pull/87 "Fixing package madule related issues."
Could you please take a look?  Thanks, in advance!!